.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,86 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report for cuDF
+title: "[BUG] "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and fill out this bug report!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open bugs](https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this request
```

Comment:
Happy to soften the language, but want to ensure we include a link to open issues of the same type both to prevent extra work on our team, and to help users who may be unfamiliar with GitHub.

We could also change the link to our project, with custom filtering to only show bugs (and grouped by roadmap to show expected resolution date if available): https://github.com/orgs/rapidsai/projects/51/views/1?filterQuery=label%3ABug%2C+is%3Aopen%2C+is%3Aissue

Reply:
I think I am $-\epsilon$ on linking to the project board instead. My guess is that that will be even less familiar.

My worry is that there are currently ~230 open issues in cudf (yes, we should perhaps do better closing things), which can be overwhelming for a flyby bug report to spelunk through.

Especially since the wording is "You confirm you have searched and found no duplicates".

How about something like

> - Please link to any related \[open bugs\](...)

TBH, for now at least, it doesn't _feel_ like we're getting lots of repeat issues.

Finally (minor), shall we use "open issues" rather than "open bugs" (since that is the terminology in the github interface).

Reply:
I like that approach because implicitly I think if people found a duplicate in that search they'd add a comment to that issue instead of filing a new report. 

I prefer "open bugs" to "open issues" because the link is pre-filtered to issues with the `bug` label. We could do "open bug issues" but IMO that's awk. Same reasoning/wording change happens in docs and feature reqs.

Reply:
Committed the softer wording to all of the templates (I left in "open bugs"). Not resolving in case you really don't like calling them "open bugs" but it's definitely my pref.